packages/core/src/utils/rpc.ts

```diff
@@ -146,6 +146,24 @@ export const _eth_getTransactionReceipt = (
       return receipt as SyncTransactionReceipt;
     });
 
+/**
+ * Helper function for "eth_getBlockReceipts" request.
+ */
+export const _eth_getBlockReceipts = (
```

Comment:
Can `_eth_getTransactionReceipts` be removed?

Reply:
Some test files are using `_eth_getTransacationReceipt`, but I will remove this method before merging